source/actors/PlatformActor.tscn

```diff
@@ -39,14 +39,16 @@ script = ExtResource( 7 )
 speed = 2400.0
 distance = 400.0
 
-[connection signal="dash_pressed" from="ActionBus" to="Dash" method="set_physics_process" binds= [ true ]]
+[connection signal="available_jumps_depleted" from="Jump" to="Dash" method="set_air_dash" binds= [ false ]]
```

Comment:
I think that two behaviours should not be directly connected, that's why we are trying to use a "Manager" right? 
Once Jump "replenished"  or "depleted" their actions, should send a signal to the Manager who will act accordingly - imagine something else happens when your jumps are depleted, like you can make a crash on the floor pressing the button down ( like the one you can do with Mario, don't know if it has a name), if this is the case you should connect multiple signal to different components and that is not elegant

Reply:
Yeah I didn't implement the manager yet. I am still trying to foresee if using the Inspector's interface can be enough. Since the manager will only tell what triggers what, it must not have conditionals etc...it would be basically an *ad hoc* implementation to explicit that signal connections.

Still wrapping my head around this manager